packages/contractkit/src/wallets/ledger-wallet.ts

```diff
@@ -301,4 +325,15 @@ export class LedgerWallet implements Wallet {
     }
     throw error
   }
+
+  private compareVersionWithTheLedgerApp(version: string): number {
```

Comment:
To make the order more explicit can you take in two string params and make this static. Intuitively I'm not sure what -1 means since the order isn't strictly defined.